spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/authenticated_user'
```

Comment:
I'm not pretty sure, but if you decomment the line 27 of `rails_helper.rb`, you will not need to add this require.https://github.com/wolox-training/jm-rails/blob/839dd8798b66c02a85cae49ced702aedb9e9c619/spec/rails_helper.rb#L27

Reply:
You're correct, fixing this...